package.json

```diff
@@ -8,7 +8,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "test:lint": "yarn eslint . --ignore-path .gitignore --ext .js",
-    "format": "prettier \"src/**/*.{js,jsx,json,css}\" --write",
+    "format": "prettier \"src/**/*.{js,jsx,ts,tsx,scss,sass,css,json}\" --write",
     "predeploy": "yarn build",
     "deploy": "gh-pages -d build",
```

Comment:
I would recommend you to use GitHub Actions to deploy instead of using this package.

Check out: https://github.com/marketplace/actions/deploy-to-github-pages :rocket:

Reply:
I'll change after this to Github Actions.